<?xml version="1.0" encoding="utf-8" standalone="no"?>
<test-run id="2" testcasecount="1" result="Failed" label="Error" total="1" passed="0" failed="1" inconclusive="0" skipped="0" asserts="0" engine-version="3.9.0.0" clr-version="4.0.30319.42000" start-time="2019-04-09 12:02:51Z" end-time="2019-04-09 12:02:51Z" duration="0.044514">
  <command-line><![CDATA[nunit3-console.exe  C:\QA\Simulation\Testing\nunit-console-master\bin\Debug\net35\nunit3-console.tests.dll]]></command-line>
  <test-suite type="Assembly" id="0" name="" fullname="" runstate="NotRunnable" testcasecount="1" result="Failed" label="Error" start-time="2019-04-09 12:02:51Z" end-time="2019-04-09 12:02:51Z" duration="0.001" total="1" passed="0" failed="1" inconclusive="0" skipped="0" asserts="0">
    <failure>
      <message><![CDATA[System.ComponentModel.Win32Exception : The parameter is incorrect]]></message>
      <stack-trace><![CDATA[--Win32Exception
The parameter is incorrect
   at System.Diagnostics.Process.GetProcessTimes()
   at System.Diagnostics.Process.get_TotalProcessorTime()
   at NUnit.Engine.Services.TestAgency.CreateRemoteAgent(TestPackage package, Int32 waitTime)
   at NUnit.Engine.Runners.ProcessRunner.CreateAgentAndRunner()
   at NUnit.Engine.Runners.ProcessRunner.RunTests(ITestEventListener listener, TestFilter filter)
]]></stack-trace>
    </failure>
  </test-suite>
</test-run>